core/tests/test_yaml_serialization.cpp

```diff
@@ -3,6 +3,9 @@
 #include <cubos/core/data/yaml_serializer.hpp>
 #include <cubos/core/data/serialization_map.hpp>
 
+#include <glm/glm.hpp>
```

Comment:
ditto

Reply:
Before it was being included in another header, but with the refactor now we have to include it directly.